### ISSUE TITLE
E2E jobs ignore the requested K8s version

### ISSUE THREAD

**Issue by skitt** [submariner-io/submariner-operator#2364]
Our E2E jobs ignore the requested K8s version and run on 1.25. This has resulted in our losing the ability to run on 1.17 without noticing.

I haven’t checked but this is presumably related to the switch from `CLUSTERS_ARGS` to `K8S_VERSION` in Shipyard.

**Comment by mkolesnik** [comment]
> I haven’t checked but this is presumably related to the switch from `CLUSTERS_ARGS` to `K8S_VERSION` in Shipyard.

Yes and no, the `K8S_VERSION` is behaving properly, the problem is when using "dapper" it only passes environment variables that were defined in `DAPPER_ENV` into the container.
We can either list all variables in the `Dockerfile.dapper` (which would be long and perhaps annoying to maintain) or just use make flags for the CI (for e2e its a simple fix, for other jobs might be more work).

For now I'll switch the jobs to use make flags, if we want to support environment variable passing we'll have to update `Dockerfile.dapper` as well.
